### PR TITLE
Add new CLI commands

### LINKED ISSUE #29667: include command line instructions for tool install
Would be incredibly helpful to have the actual instructions for installing the tool here.  Apologies if I missed them

```
$ dotnet tool install -g mlnet
You can invoke the tool using the following command: mlnet
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9e480b5e-1c4e-be32-0389-fbaa3a0c1378
* Version Independent ID: c8a49f16-d0fd-cdcf-1886-73302ea3ee53
* Content: [Automate model training with the ML.NET CLI - ML.NET](https://docs.microsoft.com/en-us/dotnet/machine-learning/automate-training-with-cli)
* Content Source: [docs/machine-learning/automate-training-with-cli.md](https://github.com/dotnet/docs/blob/main/docs/machine-learning/automate-training-with-cli.md)
* Product: **dotnet-ml**
* GitHub Login: @luisquintanilla
* Microsoft Alias: **luquinta**

### PR DESCRIPTION
**Pull request by jwood803**
## Summary

Update ML.NET CLI documentation to include `forecasting` and `train` commands.

Fixes #29667

### CODE REVIEW

**Comment by jwood803** [comment]
@luisquintanilla I believe this is ready for review

File: docs/machine-learning/automate-training-with-cli.md [hunk 1]
```diff
@@ -32,8 +32,24 @@ Currently, the ML Tasks supported by the ML.NET CLI are:
 - regression
 - recommendation
 - image classification
+- train
+- forecasting
```
Comment (luisquintanilla) [inline docs/machine-learning/automate-training-with-cli.md:36]: MInor nit, but I'd switch the order of these two.

```suggestion
- forecasting
- train
```

File: docs/machine-learning/automate-training-with-cli.md [hunk 2]
```diff
@@ -32,8 +32,24 @@ Currently, the ML Tasks supported by the ML.NET CLI are:
...
+dotnet tool install --global mlnet-<OS>-<ARCH>
+```
+
+For example, the x64 version of Windows can be installed with:
+
+```console
+dotnet tool install --global mlnet-win-x64
+```
+
+More information on installing the ML.NET CLI can be found [here](./how-to-guides/install-ml-net-cli.md).
```
Comment (luisquintanilla) [inline docs/machine-learning/automate-training-with-cli.md:50]:
```suggestion
For more information, see the [Install the ML.NET CLI](./how-to-guides/install-ml-net-cli.md) how-to guide.
```

**Review by luisquintanilla — APPROVED** [review]
Thanks for these updates @jwood803. I added a few comments. Once those are in, feel free to merge. Thanks!

**Comment by gewarren** [comment]
@jwood803 There are a couple of suggestions for you to consider.

File: docs/machine-learning/reference/ml-net-cli-reference.md [hunk 3]
```diff
@@ -163,6 +163,51 @@ mlnet recommendation
 
 Invalid input options cause the CLI tool to emit a list of valid inputs and an error message.
 
+### Forecasting options
+
+Running `mlnet forecasting` will train a time series forecasting model. Choose this command if you want an ML Model to forecast a value based on historical data (e.g. sales forecasting).
```
Comment (gewarren) [inline docs/machine-learning/reference/ml-net-cli-reference.md:168]:
```suggestion
Running `mlnet forecasting` will train a time series forecasting model. Choose this command if you want an ML Model to forecast a value based on historical data (for example, sales forecasting).
```

File: docs/machine-learning/reference/ml-net-cli-reference.md [hunk 4]
```diff
@@ -163,6 +163,51 @@ mlnet recommendation
...
+
+--train-time <train-time>
+
+-v, --verbosity <verbosity>
+
+```
+
+### Train options
+
+Running `mlnet train` will train a model based on an "mbconfig" file generated from Model Builder. For this command to work the training data will need to be in the same directory as the "mbconfig" file.
```
Comment (gewarren) [inline docs/machine-learning/reference/ml-net-cli-reference.md:201]:
```suggestion
Running `mlnet train` will train a model based on an "mbconfig" file generated from Model Builder. For this command to work, the training data must be in the same directory as the "mbconfig" file.
```

File: docs/machine-learning/tutorials/sentiment-analysis-cli.md [hunk 5]
```diff
@@ -125,17 +125,20 @@ Those enumerated assets are explained in the following steps of the tutorial.
...
+    - The generated **mbconfig file** is a config file that can be used to retrain your model, either through the CLI or through Model Builder. This will also have two code files associated with it and a zip file.
+        - The **training** file contains the code to build the model pipeline using the ML.NET API.
+        - The **consumption** file contains the code to consume the model.
+        - The **zip** file that is the generated model from the CLI.
 
-1. Open the **ModelInput.cs** and **ModelOutput.cs** class files within the class library project. You will see that these classes are 'data classes' or POCO classes used to hold data. It is 'boilerplate code' but useful to have it generated if your dataset has tens or even hundreds of columns.
+1. Open the **SampleClassification.consumption.cs** file within the **mbconfig** file. You will see that there input and output classes classes which are 'data classes' or POCO classes used to hold data. It is 'boilerplate code' but useful to have it generated if your dataset has tens or even hundreds of columns.
```
Comment (gewarren) [inline docs/machine-learning/tutorials/sentiment-analysis-cli.md:141]:
```suggestion
1. Open the **SampleClassification.consumption.cs** file within the **mbconfig** file. You'll see that there are input and output classes. These are 'data classes' or POCO classes used to hold data. The classes contain boilerplate code that's useful if your dataset has tens or even hundreds of columns.
```

File: docs/machine-learning/tutorials/sentiment-analysis-cli.md [hunk 6]
```diff
@@ -202,7 +205,7 @@ However, things got a lot easier for you than what's explained in that blog post
 
 For more advanced learning purposes, you can also explore the generated C# code that was used by the CLI tool to train the generated model.
 
-That 'training model code' is currently generated in the custom class generated named `ModelBuilder` so you can investigate that training code.
+That 'training model code' is currently generated in the file generated named `SampleClassification.training.cs` so you can investigate that training code.
```
Comment (gewarren) [inline docs/machine-learning/tutorials/sentiment-analysis-cli.md:208]:
```suggestion
That training model code is generated in the file named `SampleClassification.training.cs`, so you can investigate that training code.
```

**Review by gewarren — APPROVED** [review]

**Pull request merged**